recipes/aws-sdk-cpp/all/conandata.yml

```diff
@@ -0,0 +1,10 @@
+sources:
+  "1.8.130":
+    url: "https://github.com/aws/aws-sdk-cpp/archive/1.8.130.tar.gz"
+    sha256: "5dd09baa28d3f6f4fb03fbba1a4269724d79bcca3d47752cd3e15caf97276bda"
+patches:
+  "1.8.130":
+    - base_path: source_subfolder
+      patch_file: patches/0001-disable-sort-links
+    - base_path: source_subfolder
+      patch_file: patches/0002-force-archive-directory-to-library-directory
```

Comment:
add `.patch` extension to these patch files please, to make editorconfig and syntax highlighting happy.